packages/integrations/preact/package.json

```diff
@@ -31,6 +31,7 @@
     "dev": "astro-scripts dev \"src/**/*.ts\""
   },
   "dependencies": {
+    "@babel/core": ">=7.0.0-0 <8.0.0",
```

Comment:
Tried it and didn't work unfortunately. I think by making it a `peerDependency` you are making the user install it.

```
└─┬ @astrojs/react
  ├── ✕ missing peer @babel/core@">=7.0.0-0 <8.0.0"
  └─┬ @babel/plugin-transform-react-jsx
    ├── ✕ missing peer @babel/core@^7.0.0-0
    └─┬ @babel/plugin-syntax-jsx
      └── ✕ missing peer @babel/core@^7.0.0-0
```

Reply:
hmm, that's surprising since Astro has it as a dependency. I actually thought that having it as a dep in astro solved this, so maybe it's new behavior from pnpm? 

I guess I'm not against leaving it in deps then!